docs/source/before_start/architecture.rst

```diff
@@ -15,35 +15,35 @@ Torii
-Its mission is to send out `multisignature transactions <../core_concepts/glossary.html#multisignature-transactions>`_ that have not received enough signatures to reach the `quorum <../core_concepts/glossary.html#quorum>`_ until it is reached.
+Its mission is to send out `multisignature transactions <glossary.html#multisignature-transactions>`_ that have not received enough signatures to reach the `quorum <glossary.html#quorum>`_ until it is reached.
 
 Peer Communication Service
 --------------------------
 
-Internal component of Iroha - an intermediary that transmits `transaction <../core_concepts/glossary.html#transaction>`__ from `Torii <#torii>`__ through `MstProcessor <#MstProcessor>`_ to `Ordering Gate <#ordering-gate>`__.
+Internal component of Iroha - an intermediary that transmits `transaction <glossary.html#transaction>`__ from `Torii <#torii>`__ through `MstProcessor <#MstProcessor>`_ to `Ordering Gate <#ordering-gate>`__.
 The main goal of PCS is to hide the complexity of interaction with consensus implementation.
 
 Ordering Gate
 -------------
 
-It is an internal Iroha component (gRPC client) that relays `transactions <../core_concepts/glossary.html#transaction>`__ from `Peer Communication Service <#peer-communication-service>`__ to `Ordering Service <#ordering-service>`__.
-Ordering Gate recieves `proposals <../core_concepts/glossary.html#proposal>`_ (potential blocks in the chain) from Ordering Service and sends them to `Simulator <#simulator>`__ for `stateful validation <../core_concepts/glossary.html#stateful-validation>`__.
+It is an internal Iroha component (gRPC client) that relays `transactions <glossary.html#transaction>`__ from `Peer Communication Service <#peer-communication-service>`__ to `Ordering Service <#ordering-service>`__.
+Ordering Gate recieves `proposals <glossary.html#proposal>`_ (potential blocks in the chain) from Ordering Service and sends them to `Simulator <#simulator>`__ for `stateful validation <glossary.html#stateful-validation>`__.
 It also requests proposal from the Ordering Service based on the consensus round.
 
 Ordering Service
 ----------------
 
-Internal Iroha component (gRPC server) that receives messages from other `peers <../core_concepts/glossary.html#peer>`__ and combines several `transactions <../core_concepts/glossary.html#transaction>`__ that have been passed `stateless validation <../core_concepts/glossary.html#stateless-validation>`__ into a `proposal <../core_concepts/glossary.html#proposal>`__.
+Internal Iroha component (gRPC server) that receives messages from other `peers <glossary.html#peer>`__ and combines several `transactions <glossary.html#transaction>`__ that have been passed `stateless validation <glossary.html#stateless-validation>`__ into a `proposal <glossary.html#proposal>`__.
 Each node has its own ordering service.
 Proposal creation could be triggered by one of the following events:
 
```

Comment:
Broken link on line 54, should be `environment-specific parameters <../configure/index.html#environment-specific-parameters>`

Reply:
Same here. Fixed.